lib/plsql/spec/cli.rb

```diff
@@ -21,12 +21,8 @@ def initialize(*)
 
       desc 'init', 'initialize spec subdirectory with default ruby-plsql-spec files'
       def init
-        empty_directory 'spec'
-        %w(spec_helper.rb database.yml).each do |file|
-          copy_file file, "spec/#{file}"
-        end
-        directory 'helpers', 'spec/helpers'
         empty_directory 'spec/factories'
+        directory 'spec', 'spec'
```

Comment:
This is because the templates directory contained **the content of** `spec` directory to initialise project with.
After the change, project is simply initialised with the `spec` folder contained within the `templates`.
Additionally a file `.rspec` is added to the project root.
All the files that existed in `lib/plsql/spec/templates` were moved to `lib/plsql/spec/templates/spec`